web/src/app/pages/main-page-container/main-page/job-dialog/task-editor/_task-editor.component-theme.scss

```diff
@@ -27,13 +27,13 @@
 
   // TODO(amysorto): use typography-config once typography levels defined
 
-  .task-type {
-    font-size: 12px;
-  }
+  // .task-type {
```

Comment:
Can we remove this too?

Reply:
> Can we remove this too?

Looks like the PR is still in Draft state (work in progress)

Reply:
Ah my bad!